prep-css-flexbox/align-items.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - align-items</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+    /* your styles here 👇 */
+    div {
+      align-items: center;
+      justify-content: center;
+    }
+
+    #card-deck {
+      display: flex;
+      flex-basis: calc(100% / 3);
```

Comment:
`flex-basis` is a property that should be used on the child elements of a `display: flex` parent. Try applying it to the `.card-container` elements.